__tests__/comments.test.js

```diff
@@ -0,0 +1,59 @@
+import pool from '../lib/utils/pool.js';
+import setup from '../data/setup.js';
+import request from 'supertest';
+import app from '../lib/app.js';
+
+describe('RESTful comment routes', () => {
+  let user;
+  let agent;
+  let post;
+  let postRes;
+  beforeAll(async () => {
+    agent = request.agent(app);
+
+    user = { email: 'cabbott94@gmail.com', password: 'hello' };
+    await agent.post('/api/auth/signup').send(user);
+
+    post = {
+      photoUrl: 'www.me.com/me',
+      caption: 'look at me',
+      tags: ['sun', 'summer']
+    };
+
+    postRes = await agent.post('/api/posts').send(post);
```

Comment:
nice. creating these ahead of time is the best way to not have 1000 lines of code.